src/main/java/crazypants/enderio/machine/obelisk/inhibitor/BlockInhibitorObelisk.java

```diff
@@ -86,11 +86,11 @@ public void onEnderTeleport(EnderTeleportEvent event) {
   private boolean isTeleportPrevented(World entityWorld, double d, double f, double g) {
     if (!activeInhibitors.isEmpty()) {
       Vec3d pos = new Vec3d(d, f, g);
-      for (Entry<BlockCoord, BoundingBox> e : activeInhibitors.entrySet()) {
+      for (Entry<BlockPos, BoundingBox> e : activeInhibitors.entrySet()) {
         if (e.getValue().isVecInside(pos)) {
-          BlockCoord bc = e.getKey();
-          if (entityWorld.isBlockLoaded(bc.getBlockPos())) {
-            TileEntity te = bc.getTileEntity(entityWorld);
+          BlockPos bc = e.getKey();
+          if (entityWorld.isBlockLoaded(bc)) {
```

Comment:
Feel free to cleanup things like variable names "bc -> pos" or parameters (d,f,g -> whatever getTarget() is) while you're in there anyway. We have plenty of code that was ported 5 or 6 times already and has suffered from "minimal change" syndrome... ;)